Documentation/kubernetes-upgrade.md

```diff
@@ -0,0 +1,68 @@
+# Upgrading Kubernetes
+
+This document will describe upgrading your cluster's master and worker nodes. For general information on Kubernetes cluster management, upgrades (including more advanced topics such as major API version upgrades) see the [Kubernetes upstream documentation](http://kubernetes.io/v1.0/docs/admin/cluster-management.html).
+
+## Upgrading the Kubelet
+
+The Kubelet binary ships with CoreOS releases. As the host OS is updated, the Kubelet will be upgraded as well.
```

Comment:
Maybe expand this to say "this step will not be covered below in the master- or worker-specific documentation"